packages/babel-types/src/types.ts

```diff
@@ -0,0 +1,2518 @@
+// NOTE: This file is autogenerated. Do not modify.
```

Comment:
Could we move this file to `packages/babel-types/src/types/generated/index.ts`, similarly to how we do it for other generated files? :pray: